engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -212,7 +219,12 @@ private ComponentJob getComponentByKey(final AnalysisJob job, final String queri
         return null;
     }
 
-    public String getResultPath() {
+    /**
+     * Gets the path defined in the job properties file
+     * 
+     * @return
+     */
+    public String getResultPathUsercustomized() {
```

Comment:
And looking at what it does, compared to the method ```getResultJobFilePath``` below, then I don't think this method should be public since it serves only an internal purpose.